eng/pipelines/templates/jobs/vmr-build.yml

```diff
@@ -415,18 +415,31 @@ jobs:
         DownloadArchive "source-built SDK" "PrivateSourceBuiltSdkVersion" false "${{ parameters.artifactsRid }}" "$(sourcesPath)/prereqs/packages/archive/"
       displayName: Download Previously Source-Built SDK
 
+  - ${{ if eq(variables['System.TeamProject'], 'internal') }}:

```

Comment:
This should be unnecessary. The template doesn't do anything if the project is `public`